components/dot_menu/dot_menu.tsx

```diff
@@ -358,6 +372,21 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
         }
     }
 
+    fetchBindings = () => {
```

Comment:
We don't usually fetch in the components, can we make this a redux action? The component is already connected to redux so it will be quite easy. That way you can assign the bindings in the props directly in the dot_menu/index as well.

Reply:
The thing is that this information is not in redux, and we probably don't want to put it in redux. This would mainly work for posts in the search menu. The bindings might be on a different channel than the one we are currently at, so the information in redux may not be correct. This PR https://github.com/mattermost/mattermost-webapp/pull/8386 will add the RHS state to redux, so I will wait for that to be merged before giving the final touches to this.

Reply:
I am not sure I understand which information you are referring to. I just meant that since the `Client4.getAppsBindings` is a promise and we generally prefer to keep the UI simple and use actions for those operations, I would prefer if we create a new action for getting the correct app bindings based on location, and then just assign the appBindings to the result of that action call in the `index.ts` this will simplify the code and help with performance as the component wouldn't need to do the extra computation.

Reply:
> I would prefer if we create a new action for getting the correct app bindings based on location, and then just assign the appBindings to the result of that action call in the `index.ts`

@nevyangelova Are you saying that `index.ts` should process the return value from the action, and adjust the props going to the component? Or should the component be putting the result into its own state as the PR currently stands? Note that the intention is not to have this data in redux